base_group_backend/models/backend_dummy_model.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
```

Comment:
is it possible to use https://pypi.org/project/odoo-test-helper/ FakeModelLoader instead of creating a fake model ?